homeassistant/components/mysensors/handler.py

```diff
@@ -92,5 +93,5 @@ def _handle_child_update(hass, hass_config, validated):
 @callback
 def _handle_node_update(hass, msg):
     """Handle a node update."""
-    signal = NODE_CALLBACK.format(id(msg.gateway), msg.node_id)
+    signal = f"mysensors_node_callback_{id(msg.gateway)}_{msg.node_id}"
```

Comment:
I prefer using a constant for the strings that are used in more than one place.

Reply:
Should I revert my changes for mysensor to preserve the constants?

Reply:
Yes, please. But also in general I think it's more important that we guard against typos and bugs, which using constants does, than converting everything to f-strings. So I'd like us to keep using constants with str.format where needed.

Reply:
Sure I will do that. Oh yes that is definitely an important thing, not sure how to handle this though with this current and previous PRs that have been merged.
Should I go back and revert everything that used a constant? And should this affect only this or the previous PRs?

Reply:
That might be good, but it's not urgent I'd say.

Reply:
Okay thanks for the feedback!

Reply:
Done in https://github.com/home-assistant/core/pull/32271/commits/dfbb3323945fc207ca4bf70d5ecbe0a872bff072